crypto/aes/asm/aes-586.pl

```diff
@@ -2991,7 +2991,7 @@ ()
 
 	&xor	("eax","eax");			# return success
 &function_end("AES_set_decrypt_key");
-&asciz("AES for x86, CRYPTOGAMS by <appro\@openssl.org>");
+&asciz("AES for x86, CRYPTOGAMS by <https://github.com/dot-asm>");
```

Comment:
I think these "dog marks" should all be deleted. There's no need for these to be included in the binaries.

Reply:
Sure.  Pushed a commit to remove them.